assets/app/scripts/extensions/javaLink.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 angular
-  .module('openshiftConsoleExtensions')
+  .module('openshiftConsoleExtensions', ['openshiftConsole'])
```

Comment:
Will do the `/javalink` dir with a rename to `javaLinkExtension`, agree, that makes sense!